### ISSUE TITLE
500 when trying to embed vocabulary entries

### ISSUE THREAD

**Issue by artlowel** [DSpace/DSpace#3096]
**Describe the bug**
If you retrieve a vocabulary, and ask to embed the entries link the server returns a 500

**To Reproduce**
1. Authenticate as an admin
2. Retrieve a vocabulary and verify that it works, e.g. [common_types](https://dspace7.4science.cloud/server/#https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types)
4. Retrieve its entries link and verify that it works, e.g. [common_types/entries](https://dspace7.4science.cloud/server/#https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types/entries)
5. Try to embed the entries in the original request, and verify that it fails, e.g. [common_types?embed=entries](https://dspace7.4science.cloud/server/#https://dspace7.4science.cloud/server/api/submission/vocabularies/common_types?embed=entries)

**Expected behavior**
The request should succeed and any entries should be embedded

**Comment by tdonohue** [comment]
@artlowel : I agree this is a bug that needs resolving.  However, for scheduling/prioritization purposes, do you have an immediate use case for this?  Just trying to determine if this is a must fix in beta 5 / 7.0, or if this is something that could be slotted for 7.1.  (I'm also a little worried the implementation here might be more complex than it seems, since these vocabularies are driven by XML configuration they may need a custom embed solution....but I haven't looked at the code deeply here.)

**Comment by artlowel** [comment]
No it's not urgent, it just means that some extra requests will need to happen. I've disabled the embeds in https://github.com/DSpace/dspace-angular/pull/975 and [added TODO's](https://github.com/DSpace/dspace-angular/pull/975/files#diff-2436a28755bcfea336a37144fc846fe91b76ce9adbda03a3e3fb579ea364e89dR251) to turn them back on with a link back to this ticket.

**Comment by tdonohue** [comment]
Bug still exists in 7.5.  While it's low priority, moving to 7.6 board for tracking.
